pom.xml

```diff
@@ -15,6 +15,7 @@
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Promoted+Builds+Plugin</url>
 
   <properties>
+    <jenkins.version>1.565</jenkins.version>
     <findbugs-maven-plugin.version>3.0.3</findbugs-maven-plugin.version>
     <findbugs.failOnError>true</findbugs.failOnError>
```

Comment:
These two properties (`findbugs.failOnError`and `findbugs-maven-plugin.version`) can go away when the build section is removed.